Source/WebCore/html/EmailInputType.cpp

```diff
@@ -88,6 +91,14 @@ bool EmailInputType::supportsSelectionAPI() const
     return false;
 }
 
+void EmailInputType::attributeChanged(const QualifiedName& name)
+{
+    if (name == multipleAttr)
```

Comment:
@cdumez Should we prefer `switch (name.nodeName())` now?

Reply:
I don't think using a switch for a single check is better. My rule of thumb has been to use it when there are at least 3 checks (no perf benefit with less checks and I personally don't think it improves code readability much either).